crates/task/src/task_template.rs

```diff
@@ -33,6 +33,11 @@ pub struct TaskTemplate {
     /// Current working directory to spawn the command into, defaults to current project root.
     #[serde(default)]
     pub cwd: Option<String>,
+    /// Controls whether the task context is reevaluated prior to execution of a task.
```

Comment:
These wordings come from the old code, but let's fix them a bit anyway?

```suggestion
    /// Controls whether the task context is reevaluated prior to rerunning the task.
```